src/components/button-toggle/button-toggle.stories.mdx

```diff
@@ -0,0 +1,151 @@
+import { useState } from 'react';
+import { Meta, Story, Preview, Props} from '@storybook/addon-docs/blocks';
+import { action } from '@storybook/addon-actions';
+import ButtonToggle from '../button-toggle';
+
+<Meta title="Button Toggle" />
+
+# Button Toggle
+
+Press one of the buttons to make selection
+
+## Overview 
+
+ButtonToggle component should be used when user has to make a choice between a small number of options.
+
+## Guidance, hints and tips 
+
+- An icon could be added to make the meaning of an option clearer.
+- This component should not be used as part of the `Form`, `ButtonToggleGroup` should be used instead.
+
+## Quick Start 
+
+To use the `ButtonToggle` component you have to import the `<ButtonToggle />` component:
+
+```javascript
+import ButtonToggle from 'carbon-react/lib/components/button-toggle';
+```
+
+## Basic usage
+
+<Preview>
+  <Story name='basic' parameters={{ info: { disable: true }}} >
```

Comment:
I think we can pass `parameters={{ info: { disable: true }}}` in `Meta` to avoid duplication in every `<Story>`

Reply:
fixed